locale/en/docs/guides/anatomy-of-an-http-transaction.md

```diff
@@ -126,7 +126,7 @@ request.on('error', function(err) {
 });
 ```
 
-There are other ways of [handling these errors][] such as using [`domains`][] or
+There are other ways of [handling these errors][] such as
```

Comment:
I actually don't know. Things are a bit looser in the nodejs/nodejs.org repo than they are in the nodejs/node repo. My best guess is that if all you did was modify this guide and it's a small modification, don't worry about it too much. Make a PR to note the change and what further changes you propose.